BridgeEmulator/HueEmulator3.py

```diff
@@ -1577,20 +1585,33 @@ def read_http_request_body(self):
         return b"{}" if self.headers['Content-Length'] is None or self.headers[
             'Content-Length'] == '0' else self.rfile.read(int(self.headers['Content-Length']))
 
+    @staticmethod
+    def extract_version(line: str):
+        if match := re.search(r'Firmware( |(&nbsp;))+([0-9]+).*Bridge( |(&nbsp;))+V2', line, re.IGNORECASE):
```

Comment:
Flexible regex - case insensitive, whitespace between words can be any combination of spaces and &nbsp; codes.